examples/next-app/components/Voice.tsx

```diff
@@ -13,6 +18,51 @@ export const Voice = ({ accessToken }: { accessToken: string }) => {
         // eslint-disable-next-line no-console
         console.log('message', message);
       }}
+      onToolCall={async (
+        toolCall: ToolCall,
+      ): Promise<ToolError | ToolResponse> => {
+        // eslint-disable-next-line no-console
+        console.log('toolCall', toolCall);
+
+        if (toolCall.name === 'weather_tool') {
+          try {
+            const result = await fetch(
+              'https://httpbin.org/status/400',
+              // 'https://api.weather.gov/gridpoints/TOP/31,80/forecast',
+              {
+                method: 'GET',
+              },
+            );
+            const json = await result.json();
+            const forecast = json.properties.periods;
+
+            return {
+              type: 'tool_response',
+              tool_call_id: toolCall.tool_call_id,
+              content: JSON.stringify(forecast),
+            };
+          } catch (error) {
+            return {
+              type: 'tool_error',
+              tool_call_id: toolCall.tool_call_id,
+              error: 'Weather tool error',
+              code: 'weather_tool_error',
+              level: 'error',
+              content: 'There was an error with the weather tool',
+            };
+          }
+        } else {
+          return {
+            type: 'tool_error',
+            tool_call_id: toolCall.tool_call_id,
+            error: 'Tool not found',
+            code: 'tool_not_found',
+            level: 'warning',
+            content: 'The tool you requested was not found',
+          };
+        }
+      }}
+      configId="d9fa1297-7e9d-4b9f-87c3-f748b0b8ce5e"
```

Comment:
let's make this an env variable (e.g. `NEXT_PUBLIC_HUME_VOICE_CONFIG_ID`) so we don't have to make a code change for local testing